toast-tk-adapters-api/src/main/java/io/toast/tk/adapter/constant/AdaptersConfigProvider.java

```diff
@@ -40,8 +42,7 @@ private void initConfig() {
 					LOG.error(e.getMessage(), e);
 				}
 			}
-		}
-		try (final Reader resourceFileReader = new InputStreamReader(resourceAsStream)) {
+		}try (final Reader resourceFileReader = new InputStreamReader(resourceAsStream)) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)